### ISSUE TITLE
Archiving repositories

### ISSUE THREAD

**Issue by neoncitylights** [Brickimedia/brickimedia#477]
Making this ticket after a ping in https://github.com/Brickimedia/splash/commit/ea75cbb2fae2f61d76cf05347836a4ac54de9c87 (thanks for the ping!)

We'll probably be fine with deleting most of them or all of them. Here are some reasons why:

* Don't have a purpose anymore (e.g the LocalSettings repo or extensions repo which is just a group of gitmodules)
* others have been already migrated (e.g Refreshed and ArticleRatings)
* Migrated *and* merged into something else (NumberOfComments was merged into Comments, GlobalContribs is in progress of being merged into GlobalContributions, DeepSea was archived after it was merged into the Theme extension, becoming a new available theme)
* Archived (Snippet was archived in favor of TextExtracts extension), or USERNAME as an example, which didn't work as a PHP implementation.
* or projects that used to be worked on but aren't worked on anymore (the Answers extension, Custard + Lia + Quartz skins, design-guide)

Pinging for @georgebarnick and @mary-kate for thoughts on this.

**Comment by georgebarnick** [comment]
Is there a built-in way to archive a repository through GitHub?

**Comment by neoncitylights** [comment]
Yes. (https://help.github.com/articles/archiving-a-github-repository/ https://help.github.com/articles/archiving-repositories/ etc)

If archiving is preferred to deleting, I'm fine with that too

**Comment by georgebarnick** [comment]
I'd rather that just to preserve backlinks and other traces where we might have linked to stuff in the past.

**Comment by georgebarnick** [comment]
Archived LocalSettings and extensions as they're both deprecated repositories. Waiting to hear if anyone else has thoughts on archiving other repositories before proceeding.

**Comment by neoncitylights** [comment]
I was thinking of blanking the skin + extension repos so that just in case there's people who are using those, are forced to use whatever we provide as the replacement in the README instead. (e.g Brickimedia/Refreshed should provide a link to wikimedia/mediawiki-skins-Refreshed, Brickimedia/Snippet provides a link to wikimedia/mediawiki-extensions-TextExtracts). Thoughts on that?

**Comment by georgebarnick** [comment]
Would it be possible for us to turn Brickimedia/Refreshed into a mirror of wikimedia/mediawiki-skins-Refreshed?

**Comment by neoncitylights** [comment]
After some looking, apparently that's possible (didn't know that till now). Here's the link (which contains 3 different sections): https://help.github.com/articles/duplicating-a-repository/
